### PR TITLE
Add MaximumReceiveMessageSize option

### PR DESCRIPTION
**Pull request by BrennanConroy**
Need some word smithing!

### CODE REVIEW

File: aspnetcore/signalr/configuration.md [hunk 1]
```diff
@@ -136,6 +137,21 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env)
 
 The following table describes options for configuring ASP.NET Core SignalR's advanced HTTP options:
 
+::: moniker range=">= aspnetcore-3.0"
+
+| Option | Default Value | Description |
+| ------ | ------------- | ----------- |
+| `ApplicationMaxBufferSize` | 32 KB | The maximum number of bytes received from the client that the server buffers before applying backpressure. Increasing this value allows the server to receive larger messages more quickly without applying backpressure, but can negatively impact memory consumption. |
```
Comment (mikaelm12) [inline aspnetcore/signalr/configuration.md]: >negatively impact memory consumption

Is an interesting choice of words here lol.

Comment (halter73) [reply aspnetcore/signalr/configuration.md]: Yeah. "but can increase memory consumption" is probably a little clearer.

File: aspnetcore/signalr/configuration.md [hunk 2]
```diff
@@ -136,6 +137,21 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env)
...
+| `AuthorizationData` | Data automatically gathered from the `Authorize` attributes applied to the Hub class. | A list of [IAuthorizeData](/dotnet/api/microsoft.aspnetcore.authorization.iauthorizedata) objects used to determine if a client is authorized to connect to the hub. |
+| `TransportMaxBufferSize` | 32 KB | The maximum number of bytes sent by the app that the server buffers before applying backpressure. Increasing this value allows the server to send larger messages more quickly without applying backpressure, but can negatively impact memory consumption. |
```
Comment (halter73) [inline aspnetcore/signalr/configuration.md]: In the output case, it's not really the server applying backpressure, though I see what you're getting at. I would probably just say something like the following:

```suggestion
| `TransportMaxBufferSize` | 32 KB | The maximum number of bytes sent by the app that the server buffers before observing backpressure. Increasing this value allows the server to buffer larger messages more quickly without awaiting backpressure, but can increase memory consumption. |
```

File: aspnetcore/signalr/configuration.md [hunk 3]
```diff
@@ -136,6 +137,21 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env)
...
+| `Transports` | All Transports are enabled. | A bitmask of `HttpTransportType` values that can restrict the transports a client can use to connect. |
```
Comment (halter73) [inline aspnetcore/signalr/configuration.md]: Nit: I think the following might be more clear to your average C# dev.

```suggestion
| `Transports` | All Transports are enabled. | A bit flags enum of `HttpTransportType` values that can restrict the transports a client can use to connect. |
```

File: aspnetcore/signalr/configuration.md (hunk 1 above)
Comment (BrennanConroy) [inline aspnetcore/signalr/configuration.md]:
```suggestion
| `ApplicationMaxBufferSize` | 32 KB | The maximum number of bytes received from the client that the server buffers before applying backpressure. Increasing this value allows the server to receive larger messages more quickly without applying backpressure, but can increase memory consumption. |
```

File: aspnetcore/signalr/configuration.md (hunk 3 above)
Comment (anurse) [reply aspnetcore/signalr/configuration.md]: There's another table that also uses `bitmask`. @BrennanConroy you should update that one too.

**Comment by BrennanConroy** [comment]
@Rick-Anderson should I merge or someone on the docs team?

**Comment by Rick-Anderson** [comment]
@BrennanConroy feel free to merge after it's been approved - or ping me or anyone else on the team.

**Pull request merged**